packages/types/src/types.ts

```diff
@@ -7,3 +9,10 @@ export {ts as allForks} from "./allForks";
 
 /** Common non-spec type to represent roots as strings */
 export type RootHex = string;
+
+/** This type helps response to beacon_block_by_range and beacon_block_by_root more efficiently */
+export type ReqRespBlockResponse = {
```

Comment:
The `types` package should only contain types defined in the spec. `RootHex` is the exception since it's used in lodestar, fork-choice and validator. Since `ReqRespBlockResponse` is a type that only exist and makes sense in reqResp, I would define it in `network/reqResp`.